pkg/tagger/tagstore/entity_tags.go

```diff
@@ -43,35 +50,29 @@ func (e *EntityTags) getStandard() []string {
 }
 
 func (e *EntityTags) get(cardinality collectors.TagCardinality) []string {
-	return e.getHashedTags(cardinality).Get()
+	return e.getHashedTags(cardinality).UnsafeReadOnlySlice()
```

Comment:
Should you make a copy here as you did previously?

Reply:
It doesn't hurt!